frontend/src/modules/machinery/components/CaighdeanTranslation.js

```diff
@@ -0,0 +1,38 @@
+/* @flow */
+
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+
+type Props = {|
+    source: {
+        type: string,
```

Comment:
You don't even need to pass `type`, since you know that it has to be, in this case, a `caidhdean` source. I think you should make `props` flatter, and pass only the data the components needs directly. In this specific case, the component doesn't actually need any props! :-)

Reply:
Note that these comments of course apply to all other components as well.